Dockerfile

```diff
@@ -1,33 +1,26 @@
-FROM ubuntu:latest
-MAINTAINER ome-devel@lists.openmicroscopy.org.uk
+ARG BUILD_IMAGE=openjdk:8
 
-RUN apt-get -q update && apt-get -qy install openjdk-8-jdk \
-  maven \
-  ant \
-  git \
-  python-sphinx \
-  locales
+# Build image
+FROM ${BUILD_IMAGE}
+LABEL maintainer="ome-devel@lists.openmicroscopy.org.uk"
 
-RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
-    dpkg-reconfigure --frontend=noninteractive locales && \
-    update-locale LANG=en_US.UTF-8
+USER root
+RUN apt-get -q update && apt-get -qy install maven \
+   ant \
+   git \
+   python-sphinx
 
-ENV LANG en_US.UTF-8 
+RUN id 1000 || useradd -u 1000 -ms /bin/bash build
+COPY --chown=1000:1000 . /bio-formats-build
 
-RUN useradd -m bf
-
-COPY . /bio-formats-build
-RUN chown -R bf /bio-formats-build
-
-USER bf
+USER 1000
 WORKDIR /bio-formats-build
 RUN git submodule update --init
-RUN mvn clean install -DskipSphinxTests
+RUN mvn clean install -DskipSphinxTests -Dsurefire.useSystemClassLoader=false
```

Comment:
If I'm guessing correctly then this adjustment may not be needed for long, might be worth a card to come back and try removing it later this year. (There's an OpenJDK 8 version that's been around a bit that has one patch but not the followup and that state interacts poorly with surefire; it starts working again when both are included.)